src/app/edit-columns-modal/edit-columns-modal.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, Inject } from '@angular/core';
+import { LfLocalizationService} from '@laserfiche/lf-js-utils';
+import { ColumnDef } from '@laserfiche/lf-ui-components/lf-selection-list';
+import { MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
+
+
+const resources: Map<string, object> = new Map<string, object>([
+  ['en-US', {
+    'OK': 'Ok',
+    'CANCEL': 'Cancel',
+    'ADD_REMOVE_COLUMNS': 'Add/Remove Columns',
+  }],
+  ['es-MX', {
+    'OK': 'Ok - Spanish',
+    'CANCEL': 'Cancel - Spanish',
+    'ADD_REMOVE_COLUMNS': 'Add/Remove Columns - Spanish',
+  }]
+]);
+
+interface EditColumnsDialogData {
+  columnsSelected: ColumnDef[];
+  allColumnOptions: ColumnDef[];
+  updateColumns: (columns: ColumnDef[]) => void;
+}
+
+
+@Component({
+  selector: 'app-edit-columns-modal',
+  templateUrl: './edit-columns-modal.component.html',
+  styleUrls: ['./edit-columns-modal.component.css']
+})
+
```

Comment:
remove space on line 32 (this is an annotation on the class so should be directly above